cluster/juju/layers/kubernetes-worker/config.yaml

```diff
@@ -13,7 +13,7 @@ options:
       cluster. Declare node labels in key=value format, separated by spaces.
   allow-privileged:
     type: string
-    default: "auto"
+    default: true
     description: |
       Allow privileged containers to run on worker nodes. Supported values are
```

Comment:
We should update the description to deprecate this flag. I think we should also remove auto, but maybe that's another issue.

Reply:
@hyperbolic2346 yes - I can update the description if you wish, and I also think the auto option should be removed, but perhaps this can be another PR. 

Will you approve if I make those changes?

Reply:
I've opened https://github.com/juju-solutions/bundle-canonical-kubernetes/issues/585 to address the deprecation of `allow-privileged`. We can follow up on that later. :+1: